phantazm-zombies-mapdata/src/main/java/com/github/phantazmnetwork/zombies/map/WindowInfo.java

```diff
@@ -0,0 +1,45 @@
+package com.github.phantazmnetwork.zombies.map;
+
+import com.github.phantazmnetwork.commons.vector.Region3I;
+import net.kyori.adventure.key.Key;
+import net.kyori.adventure.sound.Sound;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+public record WindowInfo(@NotNull Region3I frameRegion,
+                         @NotNull List<String> repairBlocks,
+                         @NotNull Sound repairSound,
+                         @NotNull Sound repairAllSound,
+                         @NotNull Sound breakSound,
+                         @NotNull Sound breakAllSound) {
+    public static final Sound DEFAULT_REPAIR_SOUND = Sound.sound(
+            Key.key("minecraft:block.wood.place"),
+            Sound.Source.PLAYER,
```

Comment:
again for all of these I think BLOCK is better

Reply:
i'm pretty sure the last one is considered HOSTILE by vanilla, not BLOCK

Reply:
just the top 2 then